### DISCUSSION TITLE
Antivirus Alert in Cyberduck but not in other FTP-Client

### DISCUSSION THREAD

**Discussion by gent99** [iterate-ch/cyberduck#14316] [Q&A]
Hi,

i recorded a strange behavior while using cyberduck. While trying to download a zip file from one of our developers, our firewall anti virus protection system logged an error. After i analyzed the logs in the firewall, we came to the conclusion, that this must be an issue in cyberduck. I then installed another FTP-Client Software on the same client machine and could download the file without any issues. Using cyberduck, i could reproduce this every time. Both the developer and i checked that the file is clean and have not any virus in it. We both verified this using different endpoint security tools.

Is there a possibility to check why cyberduck produces an antivrus alert, while the other client doesn't?

Kind regards

**Comment by dkocher** [comment]
Please check with the support of your antivirus software seller.

**Comment by gent99** [comment]
we did that already, that's why i opened this issue. It's only cyberduck that's causing problems. Other ftp-clients we used to download the same file from the same server are fine.

**Comment by dkocher** [comment]
> we did that already, that's why i opened this issue. It's only cyberduck that's causing problems. Other ftp-clients we used to download the same file from the same server are fine.

The difference may be that we set a [quarantine](https://docs.cyberduck.io/cyberduck/download/#quarantine) flag `com.apple.quarantine` on the file downloaded.
